modules/eso-bitwarden-secret.adoc

```diff
@@ -16,16 +17,17 @@ The `bitwardenSecretManagerProvider` field enables the bitwarden secrets manager
 | Default
 | Validation
 
-| `enabled`
+| `mode`
 | _string_
-| `enabled` field enables the `bitwardenSecretManagerProvider`. you can set this field to `true` or `false`.
-| false
-| enum: [true false] +
+| `mode` field enables the `bitwardenSecretManagerProvider` provider state, which can be set to `Enabled' or `Disabled`. If set to `Enabled`, the Operator ensures the plugin is deployed and synchronized. If set to `Disabled`, the Bitwarden provider plugin reconciliation is disabled. The plugin and resources remain in their current state, and are not managed by the Operator.
```

Comment:
Some odd rendering issues going on with this sentence:

to `Enabled' or `Disabled`.

Looks like you used an apostrophe instead of a backtick after Enabled.